File: CS/WindowsFormsApplication82/CustomRibbonViewInfo.cs
using DevExpress.Skins;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;

namespace WindowsFormsApplication82 {
    public class CustomRibbonViewInfo : RibbonViewInfo {
        public CustomRibbonViewInfo(RibbonControl ribbon) : base(ribbon) {

        }
        protected override RibbonCaptionViewInfo CreateCaptionInfo() {
            return new CustomRibbonCaptionViewInfo(this);
        }
        CustomRibbonPageHeaderViewInfo _header;
        public override RibbonPageHeaderViewInfo Header {
            get {
                if (_header == null) {
                    _header = new CustomRibbonPageHeaderViewInfo(this);
                }
                return _header;
            }
        }
        public SkinElementInfo GetAppButtonContainerInfoEx(Rectangle bounds) {
            return GetAppButtonContainerInfo(bounds);
        }
        public class CustomRibbonPageHeaderViewInfo : RibbonPageHeaderViewInfo {
            public CustomRibbonPageHeaderViewInfo(RibbonViewInfo viewInfo) : base(viewInfo) {

            }
        }
    }
}

File: VB/WindowsFormsApplication82/CustomRibbonViewInfo.vb
Imports DevExpress.Skins
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports System
Imports System.Collections.Generic
Imports System.Drawing
Imports System.Linq
Imports System.Text

Namespace WindowsFormsApplication82
    Public Class CustomRibbonViewInfo
        Inherits RibbonViewInfo

        Public Sub New(ByVal ribbon As RibbonControl)
            MyBase.New(ribbon)

        End Sub
        Protected Overrides Function CreateCaptionInfo() As RibbonCaptionViewInfo
            Return New CustomRibbonCaptionViewInfo(Me)
        End Function
        Private _header As CustomRibbonPageHeaderViewInfo
        Public Overrides ReadOnly Property Header() As RibbonPageHeaderViewInfo
            Get
                If _header Is Nothing Then
                    _header = New CustomRibbonPageHeaderViewInfo(Me)
                End If
                Return _header
            End Get
        End Property
        Public Function GetAppButtonContainerInfoEx(ByVal bounds As Rectangle) As SkinElementInfo
            Return GetAppButtonContainerInfo(bounds)
        End Function
        Public Class CustomRibbonPageHeaderViewInfo
            Inherits RibbonPageHeaderViewInfo

            Public Sub New(ByVal viewInfo As RibbonViewInfo)
                MyBase.New(viewInfo)

            End Sub
        End Class
    End Class
End Namespace
